[16.0][MIG] edi_record_metadata_oca: Migration to 16.0

QuocDuong1306 opened the pull request.

john-herholz-dt committed:
[MIG] edi_oca: Migration to 16.0

simahawk committed:
Add edi_record_metadata_oca

Commit:
[UPD] Update edi_record_metadata_oca.pot

OCA-git-bot committed:
[ADD] icon.png

OCA-git-bot committed:
[UPD] README.rst

simahawk committed:
edi_record_metadata: improve json serializer

simahawk committed:
edi_record_metadata: store metadata before create

If an origin record is already given at creation
let's store metadata immediately before the creation really happens.
This way, if you have logic relying on metadata to be present at create
you will be able to make it work.

OCA-git-bot committed:
edi_record_metadata_oca 14.0.1.1.0

OCA-git-bot committed:
edi_record_metadata_oca 14.0.1.2.0

simahawk committed:
edi_oca: fix test class names

Having the same name for different classes prevents selection of
specific test cases when running single test suites.

simahawk committed:
edi_oca: fix edi.exchange.type.create

simahawk committed:
edi_oca: move to OCA/edi-framework

simahawk committed:
edi_oca: use queue_job__no_delay key in tests

simahawk committed:
edi_oca: drop migrations

simahawk committed:
edi_oca: drop deprecated code

OCA-git-bot committed:
[UPD] README.rst

simahawk committed:
edi: fix tests

simahawk committed:
edi: rename consumer mixin disable_edi_auto field

Commit:
[IMP] edi_record_metadata_oca: pre-commit stuff

QuocDuong1306 commented on edi_record_metadata_oca/tests/test_metadata.py:
```diff
@@ -27,6 +27,7 @@ def _setup_records(cls):
             direction="output",
         )
         cls.exc_record = cls.backend.create_record(cls.exc_type.code, {})
+        return res
```
I have changed it because of the `pre-commit`
![image](https://github.com/QuocDuong1306/edi-framework/assets/127363167/63527c36-cf07-45a4-adef-c0f12dd75e58)

Commit:
[MIG] edi_record_metadata_oca: Migration to 16.0

nilshamerlinck commented on edi_oca/__manifest__.py:
```diff
@@ -10,7 +10,7 @@
     """,
     "version": "15.0.1.5.0",
-    "website": "https://github.com/OCA/edi",
+    "website": "https://github.com/OCA/edi-framework",
```
better keep this PR for `edi_record_metadata_oca` only, can you please remove this change?

QuocDuong1306 replied:
What should I do now, if I don't change, it will break the [`pre-commit`](https://github.com/OCA/edi-framework/actions/runs/6320240070/job/17162466220?pr=18)?

nilshamerlinck replied:
ok I see, added a comment to the PR, you can ignore for now
can you please add Depends on `https://github.com/OCA/edi-framework/pull/5` in the PR's description on oca side?

QuocDuong1306 replied:
Ok, updated, thanks

nilshamerlinck committed:
edi_oca: fix _logger.debug usage

TypeError: not all arguments converted during string formatting

simahawk committed:
edi: fix access errors tests

AccessError core messages were modified

OCA-git-bot committed:
Merge PR #5 into 16.0

Signed-off-by simahawk

Commit:
[UPD] Update edi_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

simahawk committed:
edi: fix consumer get_view w/ nested forms

Select main  only as they can be nested into field custom forms.
I'm looking at you  on v16 :S

OCA-git-bot committed:
Merge PR #23 into 16.0

Signed-off-by simahawk

OCA-git-bot committed:
[BOT] post-merge updates
